state/prices/reducer.ts

```diff
@@ -1,18 +1,25 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 
 import { PricesMap } from 'sdk/types/prices';
+import { DEFAULT_QUERY_STATUS, LOADING_STATUS, SUCCESS_STATUS } from 'state/constants';
+import { FetchStatus } from 'state/types';
 
+import { fetchPreviousDayRates } from './actions';
 import { PricesState } from './types';
 
-const initialState: PricesState = {
+export const PRICES_INITIAL_STATE: PricesState = {
 	onChainPrices: {},
 	offChainPrices: {},
 	connectionError: null,
+	previousDayRates: [],
```

Comment:
I think now we've moved this to prices service it would be good to change the name from rates to prices to stay consistent.